src/status_im2/common/scan_qr_code/style.cljs

```diff
@@ -0,0 +1,155 @@
+(ns status-im2.common.scan-qr-code.style
+  (:require [quo2.foundations.colors :as colors]
+            [status-im.utils.platform :as platform]))
```

Comment:
please avoid using any code from status-im namespace where possible. This should be from `[react-native.platform :as platform]`